dataline-commons/src/main/resources/log4j2.xml

```diff
@@ -1,32 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="INFO">
+    <Properties>
+        <Property name="default-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %highlight{%p} %C{1.}(%M):%L - %X - %m%n</Property>
+    </Properties>
 
   <Appenders>
-    <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n"/>
+    <Console name="Default" target="SYSTEM_OUT">
+      <PatternLayout pattern="${default-pattern}"/>
     </Console>
+    <Routing name="Context">
+        <Routes pattern="${ctx:context}">
+            <Route key="worker">
+                <File name="${ctx:context}-${ctx:job_id}" fileName="${ctx:job_root}/logs.log">
```

Comment:
if there is no `job_root` in context does this just log to the `./logs.log`?

Reply:
I assume. we have test to protect ourself against it

Reply:
I would say that it will write to `/logs.logs`